src/VisualStudio/Core/Def/Telemetry/VSTelemetryActivityLogger.cs

```diff
@@ -17,7 +17,6 @@ internal sealed class VSTelemetryActivityLogger : ForegroundThreadAffinitizedObj
         private static readonly HashSet<FunctionId> s_functionIds
             = new HashSet<FunctionId>()
             {
-                FunctionId.NavigateTo_Search,
```

Comment:
I just read bug. I am not sure what they are asking. due to the shape of the navigate to API, there is no way for us to know lifetime of whole session. either like typing, this perf measure is moved to platform, or the way we do seems best we can.